### ISSUE TITLE
Javadoc is not beeing loaded from interfaces

### ISSUE THREAD

**Issue by MarcelCoding** [springdoc/springdoc-openapi#1548]
**Describe the bug**
If the javadoc is places at the interface and not the implementation, it isn't being loaded.

**To Reproduce**
- Spring Boot Version: `2.6.4`
- Spring Doc Version: `1.6.6`
- Spring Doc Modules: `webflux-ui`, `javadoc`

```java
public interface OAuthProvider {

  /**
   * Returns a {@link Set} with {@link Metadata} of all available OAuth provider. Included is a name witch could be
   * provided to the user as well as an id witch is used later on in the authentication flow.
   *
   * @return a {@link Set} with {@link Metadata} of all available OAuth provider.
   */
  Metadata getMetadata();
}
```

```java
@RestController
@RequestMapping("/auth/oauth")
public class OAuthControllerImpl implements OAuthController {

  // ...

  @Override
  public Set<Metadata> getMetadata() {
    return this.oAuthService.getMetadata();
  }
}
```

**Additional context**
It works if I place the javadoc at the implementation.

**Comment by bnasslahsen** [comment]
@MarcelCoding, 

This is more an enhancement request to be done in `therapi-runtime-javadoc`.
Please feel free to log a request in this [repo](https://github.com/dnault/therapi-runtime-javadoc).
And once it's covered, we will integrate it in springdoc-openapi.

**Comment by dnault** [comment]
Hi @bnasslahsen. therapi-runtime-javadoc version 0.14.0 is now available. Upgrading to this version should resolve the issue.

**Comment by dnault** [comment]
therapi-runtime-javadoc version 0.15.0 is now availble. It fixes a regression in 0.14.0. More details in the [change log](https://github.com/dnault/therapi-runtime-javadoc/blob/main/CHANGES.md).
